### PR TITLE
feat(collect-results-from-frames): add frameWaitTime option

### PR DESCRIPTION
**Pull request by isner**
Closes https://github.com/dequelabs/axe-core/issues/660

### CODE REVIEW

File: test/core/utils/collect-results-from-frames.js [hunk 1]
```diff
@@ -37,6 +37,37 @@ describe('axe.utils.collectResultsFromFrames', function () {
...
+        return orig(fn, to);
+      }
+      return 'cats';
+    };
+
+    var frame = document.createElement('iframe');
+    frame.addEventListener('load', function () {
+      var context = new Context(document);
+      var params = { frameWaitTime: 90000 };
+      axe.utils.collectResultsFromFrames(context, params, 'stuff', 'morestuff', noop,
```
Comment (marcysutton) [inline test/core/utils/collect-results-from-frames.js]: Is there a way to also cover this in an integration test, so we can make sure it works from axe.run?

Comment (dylanb) [reply test/core/utils/collect-results-from-frames.js]: specifically, across iframes

Comment (isner) [reply test/core/utils/collect-results-from-frames.js]: Good idea. I've added an integration test.

**Comment by marcysutton** [comment]
Also merged to develop-2x. Thanks Matt!

**Pull request merged**